Cargo.toml

```diff
@@ -245,6 +245,9 @@ shader_format_glsl = ["bevy_internal/shader_format_glsl"]
 # Enable support for shaders in SPIR-V
 shader_format_spirv = ["bevy_internal/shader_format_spirv"]
 
+# Enable support for transmission-related textures in the `StandardMaterial`, at the risk of blowing past the texture limit in lower-end adapters
+pbr_transmission_textures = ["bevy_internal/pbr_transmission_textures"]
```

Comment:
Should it be part of the default features?

Reply:
Unfortunately not, as it doesn't work under WebGL 😕